netutils/acl.py

```diff
@@ -175,6 +176,8 @@ def __init__(self, data: t.Any, *args: t.Any, **kwargs: t.Any):  # pylint: disab
 
         Args:
             data: A dictionary with string keys and either string or list of string values
+            args: Additional positional arguments (not used).
```

Comment:
I am confused by the "not used" comment?

Reply:
still removed that commentary since.